### DISCUSSION TITLE
Can't paste text successfully while editing in guthub mobile

### DISCUSSION THREAD

**Discussion by jakbin** [community/community#48641] [Mobile]
Can't paste text successfully while editing in guthub mobile 

https://user-images.githubusercontent.com/65591973/221420151-18e23fa5-8bae-4517-ae9d-f4079ef334e7.mp4

**Comment by midhxn** [comment]
The github app is still in build/improving stages and have less abilities and functionalities. **Better use Browser**

**Reply by jakbin** [reply]
That's why , I created this issue.
